eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -28,61 +29,73 @@ parameters:
     Linux_Python38:
       OSVmImage: 'ubuntu-18.04'
       PythonVersion: '3.8'
-
+  CloudConfigurations:
+    AzureCloud:
+      SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
 jobs:
-  - job: ${{ parameters.JobName }}
-    variables:
-      skipComponentGovernanceDetection: true
-      CoverageArg: --disablecov
+  - ${{ each cloudConfig in parameters.CloudConfigurations }}:
+    - job: ${{ parameters.JobName }}
+      variables:
+        skipComponentGovernanceDetection: true
+        CoverageArg: --disablecov
+
+      timeoutInMinutes: ${{ parameters.TestTimeoutInMinutes }}
+      strategy:
+        maxParallel: ${{ parameters.MaxParallel }}
+        matrix: ${{ parameters.Matrix }}
+      continueOnError: false
 
-    timeoutInMinutes: ${{ parameters.TestTimeoutInMinutes }}
-    strategy:
-      maxParallel: ${{ parameters.MaxParallel }}
-      matrix: ${{ parameters.Matrix }}
-    continueOnError: false
+      pool:
+        vmImage: $(OSVmImage)
 
-    pool:
-      vmImage: $(OSVmImage)
+      steps:
+        - template: /eng/common/pipelines/templates/steps/bypass-local-dns.yml
```

Comment:
This is only necessary for linux platforms, though I'm not sure if the live tests run on any other platforms yet?

Reply:
Live tests run against Windows, Mac, and Linux